release/ray_release/cluster_manager/cluster_manager.py

```diff
@@ -41,7 +41,7 @@ def set_cluster_env(self, cluster_env: Dict[str, Any]):
         self.cluster_env["env_vars"]["MATCH_AUTOSCALER_AND_RAY_IMAGES"] = "1"
         self.cluster_env["env_vars"]["RAY_gcs_storage"] = "memory"
         self.cluster_env["env_vars"]["RAY_USAGE_STATS_ENABLED"] = "1"
-        self.cluster_env["env_vars"]["RAY_USAGE_STATS_SOURCE"] = "nightly-tests"
+        self.cluster_env["env_vars"]["RAY_bootstrap_with_gcs"] = "1"
```

Comment:
Why do we delete the SOURCE env var?

Reply:
Oh just a mistake. I am still experimenting with this pr. I will get back to you once i confirm it works

Reply:
There's another subtle bug here (the hash is calculated without the env vars), I'm fixing this in the other PR (or you can add it here)

Reply:
This one: https://github.com/ray-project/ray/pull/23277/files#diff-569103799258d2583bc74456ccdb07d9b3a7b3bf8673ad9b153cd37b633a8712L49
(feel free to add to this PR as well)